dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/Limiter.mo

```diff
@@ -0,0 +1,27 @@
+within Dynawo.NonElectrical.Blocks.NonLinear;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model Limiter
+  import Modelica;
+
+  extends Modelica.Blocks.Interfaces.SISO;
+
+  parameter Real uMax "Upper limit of input signal";
+  parameter Real uMin= -uMax "Lower limit of input signal";
+
+equation
+  y = smooth(0, min(uMax, max(uMin, u)));
```

Comment:
I don't understand why you are using smooth(0,


I would have expected something like y = min(uMax, max(uMin, u)); Is there a reason ?

Reply:
I followed the example set by noEvent() documentation. I will try without smooth.

Reply:
It works well without smooth, with no impact on the results.